Formula/b/basex.rb

```diff
@@ -28,7 +28,7 @@ def install
     rm_rf "repo"
     rm_rf "data"
     rm_rf "etc"
-    prefix.install_metafiles
```

Comment:
bottle install and source build outputs the same, https://www.diffchecker.com/U0tHLqMg/